cfgov/paying_for_college/documents.py

```diff
@@ -0,0 +1,39 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+
+from paying_for_college.models import School
+from search.elasticsearch_helpers import label_autocomplete
+
+
+@registry.register_document
+class SchoolDocument(Document):
+
+    autocomplete = fields.TextField(analyzer=label_autocomplete)
+    text = fields.TextField(attr='primary_alias', boost=10)
+    nicknames = fields.TextField()
+
+    def prepare_autocomplete(self, instance):
+        alias_strings = [a.alias for a in instance.alias_set.all()]
+        nickname_strings = [n.nickname for n in instance.nickname_set.all()]
+        return alias_strings + nickname_strings
+
+    def prepare_primary_alias(self, instance):
```

Comment:
This can exist just within the fields list below.

Reply:
Also - it looks like this prepare function isn't needed as you haven't defined a `primary_alias` field on this document. You are accessing the `primary_alias` data by indexing it within the `text` field.

Reply:
Removed prepare_primary_alias